stubs/tqdm/tqdm/tk.pyi

```diff
@@ -8,7 +8,7 @@ __all__ = ["tqdm_tk", "ttkrange", "tqdm", "trange"]
 
 _T = TypeVar("_T")
 
-class tqdm_tk(Generic[_T], std_tqdm[_T]):
+class tqdm_tk(std_tqdm[_T], Generic[_T]):
```

Comment:
Technically the Generic base is redundant here, as there is another base that uses the TypeVar. That's another thing where maybe we could be more consistent.

Reply:
(Though obviously no need to change that in this PR.)